elements/lisk-cryptography/src/hash.ts

```diff
@@ -43,3 +43,13 @@ export const hash = (data: Buffer | string, format?: string): Buffer => {
 		'Unsupported data format. Currently only Buffers or `hex` and `utf8` strings are supported.',
 	);
 };
+
+export const verifyChecksum = (
+	data: Buffer | string,
+	checksum: string,
+): boolean => {
+	const dataHash = hash(data);
+	const hex = bufferToHex(dataHash);
```

Comment:
How does this work? The output of `sha256sum` looks like `48be5c5bff9fbf6077844165242856b9db6cdbf44e10cd9aeb8ae18f46acbac2  lisk-2.0.0-rc.0-Linux-x86_64.tar.gz` (notice the `\t<file name>`)

Reply:
when u generate the hex from buffer of tar file it will be just the hex `48be5c5bff9fbf6077844165242856b9db6cdbf44e10cd9aeb8ae18f46acbac2`, so it should be fine.
on the other hand, what you might be referring to `SHA256` file which has the format you mentioned, in that case i am already doing 
```
const content = fs.readFileSync(`${fileDir}/${fileName}.SHA256`, 'utf8');
const checksum = content.split(' ')[0];
```